cli-utils/setup-mcp.js

```
@@ -0,0 +1,472 @@
+
+    case 'linux':
+      paths.claudeCode = [
+        path.join(home, '.claude', '.mcp.json'), // User-level MCP config
+      ];
+      paths.claudeDesktop = [
+        path.join(home, '.config', 'Claude', 'claude_desktop_config.json'),
+      ];
+      break;
+
+    case 'win32': // Windows
+      const appData = process.env.APPDATA || path.join(home, 'AppData', 'Roaming');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)